src/components/shared/ProfileModal.tsx

```diff
@@ -247,9 +275,13 @@ const Shared = forwardRef<Ref, Props>((props, ref) => {
             style={styles.viewBtn}
           >
             <View style={styles.viewBtn}>
-              <StyledTextBtn style={{
-                color: modalBtnPrimaryFont,
-              }}>{getString('CHAT')}</StyledTextBtn>
+              <StyledTextBtn
+                style={{
+                  color: modalBtnPrimaryFont,
+                }}
+              >
```

Comment:
If you think some of the lines are wasted you can try to reduce them like in previous code.
```
<StyledTextBtn style={{
  color: ...,
 }}>{getString....
```

Reply:
resolved [5fa2ca6](https://github.com/dooboolab/hackatalk-mobile/pull/115/commits/5fa2ca6952d76e16d0ae88a7017c567c17817b4d)